src/integrations/YattIntegrationHelpers.js

```diff
@@ -26,8 +26,10 @@ export default {
     });
 
     const credential = credentials?.yatt[0];
+    let formattedData = this.formatData(data);
```

Comment:
where is `data` defined here? the `formatData`function is used with credentials are originally saved, so it shouldn't need to be called here.

Reply:
instead, create the credentials when we determine that the cookie is set in App.vue and set them to `type: "cookie"` then use that when building the headers below